test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/TemplateSandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Template test sandbox
+
+This folder exists to serve as an isolated environment for template execution tests.
+
+## Debugging template execution tests
+
+Before running template execution tests, make sure that packages defined in this solution have been packed by running the following commands from the repo root:
+```sh
+./build.cmd -build
+./build.cmd -pack
```

Comment:
Just to confirm these could be executed together - i.e., `./build.cmd -build -pack`, right?

Reply:
The one time I tried that, the dynamically generated template content (like the `.csproj` file) didn't get included in the NuGet package. So the build currently has to be run before the pack. It does seem broken, but I think it's been this way since the templates were introduced. Would be nice to fix soon 🙂

Reply:
May be worth adding a comment to mention that the commands should be run separately (until this is fixed). Because I'd be tempted to run those as one.